lib/binding_rust/bindings.rs

```diff
@@ -1,5 +1,7 @@
 /* automatically generated by rust-bindgen 0.66.1 */
 
+pub const TREE_SITTER_LANGUAGE_VERSION: u32 = 14;
+pub const TREE_SITTER_MIN_COMPATIBLE_LANGUAGE_VERSION: u32 = 13;
```

Comment:
Why u32 :raised_eyebrow:

Reply:
Because that's default type that came from `api.h` but I adapted it in https://github.com/tree-sitter/tree-sitter/pull/2540/files#diff-b3254ffc96186f12dd09ec7367f266556732e4bfc24dc84a39f966f1c73f21e3L29-R36.

Rust binding adapts `u32` to `usize` in many places but I'm not sure that it's a good approach.

Reply:
Makes sense